tachyon/base/sort.h

```diff
@@ -0,0 +1,20 @@
+#ifndef TACHYON_BASE_SORT_H_
+#define TACHYON_BASE_SORT_H_
+
+#include "third_party/pdqsort/include/pdqsort.h"
+
+namespace tachyon::base {
+
+template <typename Iter>
+void UnstableSort(Iter begin, Iter end) {
```

Comment:
Unstable sort is a category of sorting algorithms, and pdq sort is one specific algorithm within this category. Does implementing pdq sort by wrapping it with the name `UnstableSort` mean that no other unstable sorting algorithms will be implemented?

Reply:
In any programming language, a single sorting algorithm is typically chosen as the default, such as Rust and Go using pdqsort and C++ using quicksort. I believe pdqsort is better than quicksort and prefer not to have users think about the specific implementation details. This way, if a better sorting solution emerges, we can switch to it seamlessly without needing to update multiple places in the code. This choice isn't permanent and can evolve with advancements in sorting algorithms, making it easier to adapt in the future.